docs/ota-client-guide/modules/ROOT/pages/finding-unsigned-metadata.adoc

```diff
@@ -0,0 +1,25 @@
+= Find the unsigned Root and Targets metadata
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+If you want to use your own PKI you will need to know where in your local repository can you find the metadata you want to sign.
+It may be `root.json` or `targets.json` files, both are stored under `tuf/<reponame>/roles/unsigned` path.
+
+NOTE: <reponame> is the name you specified during your repository initialization with `garage-sign init`
+
+If, however, the `unsigned/` folder is empty, most likely it means you have to do a pulling first:
+
+* Use `garage-sign root pull` to pull the unsigned `root.json`
+* Use `garage-sign targets pull` to pull the unsigned `targets.json`
+
+If you haven't previously created any targets - use `garage-sign targets init` which will create the unsigned `targets.json` as well.
+
+== Generate Root and Targets metadata in canonical form
```

Comment:
This functionality is not implemented yet, right? Do you want to add it already?

Reply:
Yes, it's not implemented yet, but as per [this comment](https://saeljira.it.here.com/browse/OTA-5253?focusedCommentId=4816454&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4816454) it was decided to add it to documentation.